test/multiple-logger-test.js

```diff
@@ -24,9 +24,10 @@ define(['test/test-helpers'], function(testHelpers) {
                 expect(newLogger.methodFactory).toBeDefined();
             });
 
-            it("returns loggers without `getLogger()` and `noConflict()`", function(log) {
+            it("returns loggers without `noConflict()`", function(log) {
                 var newLogger = log.getLogger("newLogger");
-                expect(newLogger.getLogger).toBeUndefined();
+                // Now expect getLogger to be defined for the child
+                // expect(newLogger.getLogger).toBeUndefined();
```

Comment:
I think you can just remove this entirely; it’s totally irrelevant for v2. Is there an important reason to preserve the old behavior as a comment here?